protocol-units/bridge/cli/src/main.rs

```diff
@@ -0,0 +1,3 @@
+fn main() {
```

Comment:
Was this meant to be left in?

Reply:
Hey Liam,  this was intentionally included. Think of it as a temporary placeholder for the to be developed bridge command line interface.